client/web/src/search/home/QueryExamplesHomepage.tsx

```diff
@@ -85,17 +99,83 @@ export const QueryExamplesHomepage: React.FunctionComponent<QueryExamplesHomepag
+                                                    <Link target="blank" to="/help/code_search/reference/queries">
+                                                        Complete query reference{' '}
+                                                        <Icon role="img" aria-label="Open in a new tab" svgPath={mdiOpenInNew} />
+                                                    </Link>
+                                                </small>
+                                            )}
+                                        </div>
+                                    ))}
+                                </div>
+                            </TabPanel>
+                            <TabPanel>
+                                <div className={styles.queryExamplesSectionsColumns}>
+                                    {exampleQueryColumns.map((column, index) => (
+                                        // eslint-disable-next-line react/no-array-index-key
+                                        <div key={`column-${index}`}>
+                                            {column.map(({ title, queryExamples }) => (
+                                                <QueryExamplesSection
+                                                    key={title}
+                                                    title={title}
+                                                    queryExamples={queryExamples}
+                                                    onQueryExampleClick={onQueryExampleClick}
+                                                />
+                                            ))}
+                                            {!!index && (
+                                                <small className="d-block">
+                                                    <Link target="blank" to="/help/code_search/reference/queries">
+                                                        Complete query reference{' '}
+                                                        <Icon role="img" aria-label="Open in a new tab" svgPath={mdiOpenInNew} />
+                                                    </Link>
+                                                </small>
+                                            )}
+                                        </div>
+                                    ))}
+                                </div>
+                            </TabPanel>
+                        </TabPanels>
+                    </Tabs>
+                    <div className="d-flex align-items-center justify-content-lg-center my-5">
+                        <H4 className={classNames('mr-2 mb-0 pr-2', styles.proTipTitle)}>Pro Tip</H4>
+                        <Link to="https://signup.sourcegraph.com/" onClick={() => eventLogger.log('ClickedOnCloudCTA')}>
+                            Use Sourcegraph to search across your team's code.
+                        </Link>
                     </div>
```

Comment:
This doesn't look aligned to me, might need some extra CSS magic here:

```suggestion
                    <div className="d-flex align-items-baseline justify-content-lg-center my-5">
                        <H4 className={classNames('mr-2 mb-0 pr-2', styles.proTipTitle)}>Pro Tip</H4>
                        <Link to="https://signup.sourcegraph.com/" onClick={() => eventLogger.log('ClickedOnCloudCTA')}>
                            Use Sourcegraph to search across your team's code.
                        </Link>
                    </div>
```